.circleci/config.yml

```diff
@@ -555,21 +488,23 @@ workflows:
           filters:
             branches:
               only: /^release\/.*$/
-      - deploy_release_stage:
+      - deploy:
           requires:
             - hold
+          target: test
+          refreshDb: "1"
           filters:
             branches:
               only: /^release\/.*$/
       - crawl:
-          requires: [deploy_release_stage]
+          requires: [deploy]
           target: test
           samples: 700
           filters:
             branches:
               only: /^release\/.*$/
       - backstop:
-          requires: [deploy_release_stage]
+          requires: [deploy]
           target: stage
```

Comment:
I just noticed the target for backstop should be `test` missed in my first check.